getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>Add the to the beginning: "The Y-gate". Ditto below for other gates.</p><p> </p><p>Also add a period to the end of the last sentence after the matrix.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>